common/djangoapps/django_comment_common/migrations/0008_role_user_index.py

```diff
@@ -12,6 +12,6 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.RunSQL(
-            'CREATE INDEX dcc_role_users_user_role_idx ON django_comment_client_role_users(user_id, role_id);'
-        ),
+            'CREATE INDEX dcc_role_users_user_role_idx ON django_comment_client_role_users(user_id, role_id);',
+            reverse_sql='DROP INDEX dcc_role_users_user_role_idx ON django_comment_client_role_users;'),
```

Comment:
This looks reasonable to me based on my understanding of [the `RunSQL` docs](https://docs.djangoproject.com/en/3.0/ref/migration-operations/#runsql) though I am curious if you should add sql= in front of line 15 and if square brackets are needed.

I am also curious if there was a reason that @iloveagent57 didn't use [`migrations.AddIndex`](https://docs.djangoproject.com/en/3.0/ref/migration-operations/#addindex) initially and what would happen if we were to change the operations block (though I believe it's a bad idea to change forward migrations in general).

This is an example of a similar migration: https://github.com/edx/edx-platform/blob/master/openedx/features/course_duration_limits/migrations/0007_auto_20190311_1919.py#L36

Reply:
At this point, I don't remember.  From the context in https://github.com/edx/edx-platform/pull/18858 and (more importantly) the related ticket https://openedx.atlassian.net/browse/EDUCATOR-3374, I think the addition of this index ended up being unimportant and did not ultimately contribute to the resolution of the issue it meant to address.  As long as you've been able to reverse and re-apply this in a dev environment, I'm comfortable with it.